api/test_auth_endpoints.sh

```
@@ -0,0 +1,75 @@
+export AUTH0_AUDIENCE="bocchi-the-map-api"
+export AUTH0_CLIENT_ID="test-client-id-12345"
+export AUTH0_CLIENT_SECRET="test-client-secret-67890"
+export TIDB_PASSWORD="dummy-password-for-testing"
+export ENV="development"
+export LOG_LEVEL="INFO"
+export PORT="8080"
+
+# Build the API server
+echo "Building API server..."
+go build -o bin/api ./cmd/api/main.go
+if [ $? -ne 0 ]; then
```

Shellcheck (0.10.0)

[style] Check exit code directly with e.g. 'if ! mycmd;', not indirectly with $?.

(SC2181)